tests/pytest/oauth/test_views.py

```diff
@@ -30,6 +32,17 @@ def test_login(mocker, app_request):
     assert not session.logged_in(app_request)
 
 
+def test_login_analytics(mocker, app_request):
```

Comment:
Should I be using `@pytest.mark.request_path(...` here? I'm confused on what that is actually meant to do or express.

Reply:
Nah, shouldn't matter unless the request path is used somewhere.

Reply:
If you want (need) the `app_request` to point to a specific URL, it's a [mark supported by that fixture](https://github.com/cal-itp/benefits/blob/dev/tests/pytest/conftest.py#L20)

Not needed if the test doesn't "care" about the request path.